runtime/Cargo.toml

```diff
@@ -1,331 +1,288 @@
 [build-dependencies]
-substrate-wasm-builder = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate" }
+substrate-wasm-builder = {branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate"}
 
```

Comment:
You seem to have autoformatted this file into oblivion. These changes are not consistent with the standard `.toml` style.